index.html

```diff
@@ -16,13 +16,42 @@
     <nav class="nav">
       <ul class="nav__list">
         <li class="nav__item">
-          <a href="#" class="nav__link">Rules</a></li>
+          <a href="#" class="nav__link">Rules</a>
+          
+        </li>
         <li class="nav__item">
-          <a href="#" class="nav__link">Previous moves</a>
+          <a href="#" class="nav__link">Previous moves</a></li>
         </li>
         <li class="nav__item">
           <a href="#" class="nav__link">Author</a></li>
       </ul>
+      <div class="modal-bg">
+        <div class="modal">
+          <div class="modal__content">
+            <h2 class="modal__header">Rules</h2>
```

Comment:
I think the class name should be `modal__heading` here (https://wikidiff.com/heading/header)